docs/src/index.md

```diff
@@ -0,0 +1,36 @@
+# TestImages.jl
+
+This package provides a convenient Julia interface for loading
+[standard named test images](https://en.wikipedia.org/wiki/Standard_test_image) and example images for the internal usage in JuliaImages.
+This can be used in conjunction with the
+[Images](https://github.com/JuliaImages/Images.jl) package.
+
+## Installation
+```
+pkg> add TestImages
+```
+
+### Linux and OSX
+If you find yourself missing most of the images described below, please try `Pkg.build("TestImages")`, which should trigger another attempt to download the standard images.
+
+In case you would like to download other images from the repository not in the standard set, you can call the `testimage` with the image name and it will be downloaded from the repository.
+
+### Windows
+The `download` command, used to download images from the archives, is not fully supported.
+You can manually download the files listed in `deps\build.jl` from the `images` folder of the `master` branch of this repository and place them in `TestImages\images`.
```

Comment:
I just copied from the current document, but I think this is old infomation because we don't have images in `TestImages/images` directry currently. (I don't have windows environment for now, so I can't check this description.)